build/ci/e2e/GKE_k8s_versions.jenkinsfile

```diff
@@ -62,7 +62,8 @@ pipeline {
     post {
         unsuccessful {
             script {
-                def msg = "E2E tests failed!\r\n" + env.BUILD_URL
+                def msg = "E2E tests in GKE for K8s versions failed! \r\n" +
```

Comment:
"in GKE for k8s versions" sounds a bit weird. This is just our normal E2E tests pipeline, isn't it? I think I'd be in favour of keeping the existing version, this new content does not add much value?

Reply:
I see, let's close this one then